content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -34,170 +34,45 @@ To learn about the Kubernetes networking model, see [here](/docs/concepts/servic
-
-[NSX-T Container Plug-in (NCP)](https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) provides integration between NSX-T and container orchestrators such as Kubernetes, as well as integration between NSX-T and container-based CaaS/PaaS platforms such as Pivotal Container Service (PKS) and OpenShift.
-
-### OVN (Open Virtual Networking)
-
-OVN is an opensource network virtualization solution developed by the
-Open vSwitch community.  It lets one create logical switches, logical routers,
-stateful ACLs, load-balancers etc to build different virtual networking
-topologies.  The project has a specific Kubernetes plugin and documentation
-at [ovn-kubernetes](https://github.com/openvswitch/ovn-kubernetes).
-
-### Weave Net from Weaveworks
-
-[Weave Net](https://www.weave.works/oss/net/) is a
-resilient and simple to use network for Kubernetes and its hosted applications.
-Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-plugin/)
-or stand-alone.  In either version, it doesn't require any configuration or extra code
-to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
+* [Amazon AWS VPC CNI plugin](https://github.com/aws/amazon-vpc-cni-k8s)
+* [Amazon Calico for network policy enforcement](https://docs.aws.amazon.com/eks/latest/userguide/calico.html).
+* [Antrea](https://github.com/vmware-tanzu/antrea)
+* [Cilium](https://github.com/cilium/cilium) 
+* [Cisco Application Centric Infrastructure](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/application-centric-infrastructure/index.html)
+* [Coil](https://github.com/cybozu-go/coil)
+* [Contiv-VPP](https://contivpp.io/)
+* [Docker macvlan networks](https://blog.oddbit.com/post/2018-03-12-using-docker-macvlan-networks/)
+* [Flannel](https://github.com/flannel-io/flannel#flannel)
+* [Huawei CNI-Genie](https://github.com/cni-genie/CNI-Genie)
+* [Hybridnet](https://github.com/alibaba/hybridnet)
+* [Jaguar](https://gitlab.com/sdnlab/jaguar)
+* [Knitter](https://github.com/ZTE/Knitter/)
+* [Kube-OVN](https://github.com/alauda/kube-ovn)
+* [Kube-router](https://github.com/cloudnativelabs/kube-router)
+* [Microsoft Azure Virtual Network CNI plugin (hosted)](https://docs.microsoft.com/en-us/azure/virtual-network/container-networking-overview)
+* [Microsoft Azure Virtual Network CNI plugin](https://github.com/Azure/azure-container-networking/blob/master/docs/cni.md)
+* [Multus (a Multi Network plugin)](https://github.com/k8snetworkplumbingwg/multus-cni)
+* [Nodus](https://github.com/akraino-edge-stack/icn-nodus)
```

Comment:
Where did you refer this item `Nodus`?

Reply:
I found the plugins by following current link to [OVN4NFV-K8s-Plugin](https://github.com/opnfv/ovn4nfv-k8s-plugin). OVN4NFV-K8s is deprecated, they refer to Nodus.

Reply:
Thank you to detailed explanation. :smile: